magellan-sample/src/main/java/com/wealthfront/magellan/sample/IntroStep.kt

```diff
@@ -1,16 +1,29 @@
 package com.wealthfront.magellan.sample
 
-import android.content.Context
-import com.wealthfront.magellan.core.Step
-import com.wealthfront.magellan.sample.databinding.IntroBinding
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import com.example.magellan.compose.ComposeStep
 
 internal class IntroStep(
   private val goToLearnMore: () -> Unit
-) : Step<IntroBinding>(IntroBinding::inflate) {
+) : ComposeStep() {
 
-  override fun onShow(context: Context, binding: IntroBinding) {
-    binding.learnMore.setOnClickListener {
-      goToLearnMore()
-    }
+  @Composable
+  override fun Compose() = IntroStepContent(goToLearnMore)
+}
+
+@Composable
+internal fun IntroStepContent(goToLearnMore: () -> Unit) {
+  Button(onClick = goToLearnMore) {
+    Text("Go to learn more")
   }
 }
+
+@Preview(device = Devices.PIXEL_3, name = "Intro step")
+@Composable
```

Comment:
We should probably encourage the pattern on annotating the compose function with the preview annotation so that you can look at how the entire step view will look like and if we want to look at other smaller functions we can do that too. Creating a separate method just for the preview might create misdirection when reading the code.